### DISCUSSION TITLE
Text-only Reddit posts

### DISCUSSION THREAD

**Discussion by annoxes** [glanceapp/glance#20] [General]
On Reddit, you have the option to refine your search by inputting "self:yes", which allows you to exclusively return text posts while excluding any images or videos. This would become particularly handy when using the "vertical-list" layout style, as images there are unused.

**Comment by svilenmarkov** [comment]
Hey, thanks for the suggestion! 

I'll play around with Reddit's API and see if I can get it to do this properly since so far I've not been able to get this endpoint to only return results for the specified subreddit. 

https://www.reddit.com/r/selfhosted/search.json?q=self%3Ayes

**Comment by annoxes** [comment]
Sorry for the late response!

It's also possible include "subreddit:name" in the search bar to limit the results only to the specified sub. Modifying the link to include that: 

https://www.reddit.com/r/selfhosted/search.json?q=self%3Ayes+subreddit%3Aselfhosted

Only text posts from r/selfhosted :)

**Comment by svilenmarkov** [comment]
That's awesome, thanks for letting me know! I'll get it added in the next release.
